DANS-KNAW/dd-workflow-step-add-nbn-link
# Initial implementation

Fixes DD-777

# Description of changes
* A first version of the workflow step that simply puts the NBN in a description. 

# How to test
1. Deploy with https://github.com/DANS-KNAW/dd-dtap/pull/174
2. Create and publish a dataset version and observe that the NBN is added as a description element.

# Related PRs
* https://github.com/DANS-KNAW/dd-dtap/pull/174

# Notify
@DANS-KNAW/dataversedans
